app/models/dossier.rb

```diff
@@ -101,12 +105,12 @@ def retrieve_all_piece_justificative_by_type(type)
   end
 
   def build_default_champs
-    procedure.types_de_champ.each do |type_de_champ|
-      ChampPublic.create(type_de_champ_id: type_de_champ.id, dossier_id: id)
+    procedure.types_de_champ.all.each do |type_de_champ|
```

Comment:
Ça c'est le seul truc sur lequel je suis pas 100 % certain

Reply:
j'ai essayé en rails console et je ne vois pas de différences.
```
p = Procedure.first
p.types_de_champ.each { |t| t.id }
p.reload
p.types_de_champ.all.each { |t| t.id }
```
les 2 me sortent
`SELECT "types_de_champ".* FROM "types_de_champ" WHERE "types_de_champ"."type" IN ('TypeDeChampPublic') AND "types_de_champ"."procedure_id" = $1`